File: JKTogether/src/main/java/com/jk/guess/GameFor.java
package com.jk.guess;

import java.util.Random;
import java.util.Scanner;

public class GameFor {
    public static void main(String[] args) {
        int secret = new Random().nextInt(10) + 1;
        System.out.println("secret number is : " + secret);
        Scanner scanner = new Scanner(System.in);
        for(int i = 1; i <= 5; i++){
            System.out.println("Pleas enter a number (" + i + "/4)");
            int number = scanner.nextInt();
            System.out.println("the" + i + "times" + number);
            if(number > secret)
                System.out.println("lower");
            else if(number < secret)
                System.out.println("higher");
            else {
                System.out.println();
                break;
            }
        }
        System.out.println("Finish");
    }
}

File: JKTogether/src/main/kotlin/com/jk/GameFor.kt
package com.jk

import java.util.*

fun main() {
    val secret = Random().nextInt(10) +1
    println(secret)

    val scanner = Scanner(System.`in`)
    for (i in 1..4){
        print("Please enter a number: ")
        var number = scanner.nextInt()
        println("第 $i 次, $number")
        when{
            number > secret -> println("lower")
            number < secret -> println("higher")
            else -> return
        }
    }
}